src/LodeRunner/src/Services/LodeRunnerService.cs

```diff
@@ -198,8 +226,13 @@ public ITestRunService GetTestRunService()
         public void LogStatusChange(object sender, ClientStatusEventArgs args)
         {
             // TODO Move to proper location when merging with DAL
-
-            Console.WriteLine($"{args.Message} - {args.LastUpdated:yyyy'-'MM'-'dd'T'HH':'mm':'ss.fffffffK}"); // TODO fix LogStatusChange implementation
+            _ = Common.SetRunTaskClearNgsaLoggerIdValues(config, async () =>
+                {
+                    await Task.Run(() =>
```

Comment:
Seems like we should be able to simplify this repeated call by encapsulating it in an async method.  Maybe an extension method for all objects or a base class with a virtual.  The block appears in many different classes.  So, the solution would need to work for all of them.

Reply:
good call